jib-core/src/main/java/com/google/cloud/tools/jib/image/ReproducibleLayerBuilder.java

```diff
@@ -115,6 +115,34 @@ public Blob build() {
       entry.setMode((entry.getMode() & ~0777) | layerEntry.getPermissions().getPermissionBits());
       entry.setModTime(layerEntry.getModificationTime().toEpochMilli());
 
+      entry.setUserId(0);
+      entry.setGroupId(0);
+      entry.setUserName("");
+      entry.setGroupName("");
+      if (!layerEntry.getOwnership().isEmpty()) {
```

Comment:
Seems like there could be some value in separating this out for readability (and documentability)